webbpsf/optics.py

```diff
@@ -1493,9 +1510,31 @@ def __init__(self, instrument, **kwargs):
             outside=0
         )
         self.defocus_zern = basis[3]
+        self.tilt_zern = basis[2]
 
         
     def get_opd(self, wave):
+        """
+        Parameters
+        ----------
+        wave : float or obj
+            either a scalar wavelength (meters) or a Wavefront object
+        """
+        
+        if isinstance(wave, poppy.Wavefront):
+            wavelength = wave.wavelength
+        else:
+            wave = poppy.Wavefront(wavelength=float(wave))
+            wavelength = wave.wavelength
+
+        # Check for coronagraphy
+        pupil_mask = self.instrument._pupil_mask
+        if (pupil_mask is not None) and ( ('LYOT' in pupil_mask.upper()) or ('MASK' in pupil_mask.upper()) ):
```

Comment:
[optional change]
Perhaps let's rename this `is_nrc_coron` for consistency with the same variable as above. And again can be defined  in a one-liner:
`is_nrc_coron = (pupil_mask is not None) and ( ('LYOT' in pupil_mask.upper()) or ('MASK' in pupil_mask.upper()) )`